app/models/cart.rb

```diff
@@ -1,5 +1,11 @@
-class Cart < ActiveRecord::Base
-  belongs_to :user
-  has_many :items
+class Cart
+  def initialize(cart)
+    @cart = cart
+  end
 
+  def items
+    @cart.map do |k,v|
```

Comment:
Unused block argument - `v`. If it's necessary, use `_` or `_v` as an argument name to indicate that it won't be used.<br>Space missing after comma.